src/pkg/caendr/caendr/services/database_operation.py

```diff
@@ -20,51 +16,6 @@
 
 
 
-DB_OPS = {
```

Comment:
Names and titles moved to `models/datastore/DatabaseOperation.py` in the `DbOp` enum, required files moved to the `DatabaseOperationPipeline` class as part of the preflight check in `parse`